src/celldega/clust/__init__.py

```diff
@@ -98,6 +98,66 @@ def hc(df, filter_N_top=None, norm_col='total', norm_row='zscore'):
 
   return network
 
+def clustering(path_landscape_files, segmentation_parameters, cbg):
```

Comment:
This should be renamed something like `calc_cluster_signatures` since this method is calculating the "gene expression signatures" of the cell clusters.